insights/formats/text.py

```diff
@@ -2,7 +2,11 @@
 from pprint import pprint
 from insights import dr, datasource, rule
 from insights.core.context import ExecutionContext
-from insights.formats import Formatter
+from insights.formats import Formatter, render
+from colorama import Fore, Style
```

Comment:
Maybe we should do something like this for the import so it works even if colorama isn't installed:
```python
from __future__ import print_function                                           
import six                                                                      
try:                                                                            
    from colorama import Fore, Style                                            
except ImportError:                                                             
    print("Install colorama for console colors.")                               
                                                                                
    class Default(type):                                                        
        def __getattr__(*args):                                                 
            return ""                                                           
                                                                                
    class Fore(six.with_metaclass(Default)):                                    
        pass                                                                    
                                                                                
    class Style(six.with_metaclass(Default)):                                   
        pass
```

Reply:
Ok, so should the option of installing colorama to get colors be at the users discretion and the colorama dependency in setup.py be removed?